packages/providers/mgt-msal-provider/src/mgt-msal-provider.ts

```diff
@@ -99,6 +99,17 @@ export class MgtMsalProvider extends MgtBaseProvider {
     return true;
   }
 
+  /**
+   * The base URL that should be used in the graph client config.
+   *
+   * @memberof MgtMsalProvider
+   */
+  @property({
+    attribute: 'base-url',
+    type: String
+  })
+  public baseUrl: GraphEndpoint;
```

Comment:
What's the rationale for declaring the `baseUrl` property and `base-url` attribute on each provider rather than putting it onto the `MgtBaseProvider` component?